src/local_pathfinding/land/generate_land_data.py

```diff
@@ -0,0 +1,70 @@
+import csv
+import os
+import pickle
+from os.path import normpath
+from typing import Any
+
+import geopandas as gpd
+import pyproj
+from shapely.geometry import LineString, MultiPolygon, Polygon
+from shapely.ops import split
+
+
+def dump_pkl(object: Any, file_path: str):
+    os.makedirs(os.path.dirname(file_path), exist_ok=True)
+    with open(file_path, "wb") as f:
+        pickle.dump(object, f, protocol=pickle.HIGHEST_PROTOCOL)
+
+
+def load_pkl(file_path: str) -> Any:
+    with open(file_path, "rb") as f:
+        return pickle.load(f)
+
+
+def pickle_land():
```

Comment:
Should add a docstring for this function

Reply:
done